includes/admin.form.inc

```diff
@@ -96,6 +104,20 @@ function islandora_manuscript_admin_settings_form(array $form, array &$form_stat
   return system_settings_form($form);
 }
 
+/**
+ * Check if the form element is parseable by strtotime
+ *
+ * @param array $element
+ *   The element to check.
+ * @param array $form_state
+ *   The Drupal form state.
+ */
+function islandora_manuscript_validate_strtotime($element, &$form_state) {
+  if (empty($element['#value']) || !is_numeric(strtotime($element['#value'])) || strtotime($element['#value']) < 0) {
```

Comment:
After reviewing this function, we should ensure that the form element allows for a user to supply blank input

Reply:
Resolved in 336029d